### PR TITLE
Truncate copybook names #175

### PR DESCRIPTION
**Pull request by ishche**

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export interface DependenciesDesc {
 export function loadDepFile(depFileUri: vscode.Uri): DependenciesDesc {
     const copybooks: string[] = fs.readFileSync(depFileUri.fsPath).toString().split("\n")
         .filter(e => e.trim().length > 0)
-        .map(e => e.trim());
+        .map(e => e.trim().substring(0, 8));
```
Comment (sergiuilie) [inline clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts]: nice, we should also throw an error from LSP side once the user tries to put copybook name bigger than 8 chars, what do you think?

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts]: @abdrabrcom is working on it

Comment (ishche) [reply clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts]: The LSP should show a warning, the extension should truncate silently AFAIK.

Comment (zacanbrcom) [reply clients/cobol-lsp-vscode-extension/src/services/DependencyService.ts]: yesterday we agreed about **severity:warning**

**Pull request merged**